pkg/ibmvpc/examples/out/analysis_out/acl_testing3_all_vpcs_.md

```diff
@@ -1,4 +1,4 @@
-# Connectivity for VPC test-vpc1-ky
+# Endpoint connectivity for VPC test-vpc1-ky
 ## Endpoint connectivity report
```

Comment:
Can we avoid this duplication as well?

Reply:
sure, do you want to leave first line or sec line?

Reply:
First line